.github/workflows/merge-develop-to-stage.yml

```diff
@@ -84,22 +84,10 @@ jobs:
       slack_webhook_url: ${{ secrets.SLACK_WEBHOOK }}
       HOSTED_ZONE_ID: ${{ secrets.HOSTED_ZONE_ID_DEMO }}
       DOMAIN_NAME: ${{ secrets.DOMAIN_NAME_DEMO }}
-
-  deploy-example-app:
```

Comment:
maybe just remove (I agree with Samir to actually comment not delete) in the release workflow. The develop workflow can be refactored in the later ticket to move wbc to rsw. so don't edit this file _yet_